config/import-control-test.xml

```diff
@@ -20,7 +20,8 @@
 
   <!-- Till https://github.com/checkstyle/checkstyle/issues/6916 -->
   <subpackage name="checks">
-    <subpackage name="(blocks|coding|design|header|imports|indentation|javadoc)" regex="true">
+    <allow pkg="org.junit" local-only="true"/>
+    <subpackage name="(coding|design|header|imports|indentation|javadoc)" regex="true">
```

Comment:
Shouldn't this have only changed blocks package? Checks was another PR.

Reply:
rebased and resolved